com.sap.cloud.lm.sl.mta/src/test/java/com/sap/cloud/lm/sl/mta/resolvers/v3_1/DescriptorPlaceholderResolverTest.java

```diff
@@ -0,0 +1,86 @@
+package com.sap.cloud.lm.sl.mta.resolvers.v3_1;
+
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import com.sap.cloud.lm.sl.common.util.Callable;
+import com.sap.cloud.lm.sl.common.util.JsonUtil;
+import com.sap.cloud.lm.sl.common.util.TestUtil;
+import com.sap.cloud.lm.sl.mta.MtaTestUtil;
+import com.sap.cloud.lm.sl.mta.handlers.v3_1.ConfigurationParser;
+import com.sap.cloud.lm.sl.mta.handlers.v3_1.DescriptorParser;
+import com.sap.cloud.lm.sl.mta.model.SystemParameters;
+import com.sap.cloud.lm.sl.mta.model.v2_0.Platform;
+import com.sap.cloud.lm.sl.mta.model.v2_0.Target;
+import com.sap.cloud.lm.sl.mta.model.v3_1.DeploymentDescriptor;
+import com.sap.cloud.lm.sl.mta.resolvers.ResolverBuilder;
+
+@RunWith(value = Parameterized.class)
+public class DescriptorPlaceholderResolverTest {
```

Comment:
Can you extend com.sap.cloud.lm.sl.mta.resolvers.v2_0.DescriptorPlaceholderResolverTest  in order to avoid code duplication? I mean initialize() function for example. Check other tests that extends each other, e.g. DescriptorParserTest